docs/src/manual.md

```diff
@@ -13,4 +13,5 @@ BSplineApprox
 CubicHermiteSpline
 PCHIPInterpolation
 QuinticHermiteSpline
+looks_linear
```

Comment:
Make this into a different docs block like: 

````
# Utility Functions

```@docs
DataInterpolations.looks_linear
```
````

As `looks_linear` is not exported, need to use module name.

Reply:
Thanks, will change it in the next commit